Handle child elements in the click callback

JeppeJ opened the pull request:
to make images and bold italic undeline text etc. clickable in buttons

JeppeJ committed:
Handle child elements in the click callback

to make images and bold italic undeline text etc. clickable in buttons

JeppeJ commented:
Will fix: https://github.com/Peekobot/peekobot/issues/6#issue-573596771

rosswintle commented:
Hi @JeppeJ 

Thanks again for this. I'm making some updates to Peekobot tonight. PLEASE forgive me as I don't do much open source so I'm not entirely sure I'll add your contribution correctly.

I also can't see `Contributors.md` as a standard file. Is that documented somewhere? I'll add you to the README though.

Bear with me!

rosswintle committed:
Merge branch 'master' into master

rosswintle committed:
Remove Contributors.md

Forgive me if I get this wrong - I don't do much open source. I can't see `Contributors.md` documented as a standard, so I'll add you to the `README`

Thanks so much for your contribution!

rosswintle commented:
Closes #6
